src/themes/utils/generateTheme.ts

```diff
@@ -44,10 +44,10 @@ const resolveThemeValues = (themeValues: Theme, themeTokens: object) => (nodeVal
   const path = nodeValue.replace(PARENTHESES, '').split('.')
   const { $value } = get(themeValues, path) || {}
 
-  if (!$value) {
+  if ($value === undefined) {
```

Comment:
why broading the check to allow for falsey values (e.g. null)? is this necessary to allow for `0`?

Reply:
exactly